src/pages/studyView/table/ClinicalTable.tsx

```diff
@@ -125,13 +125,26 @@ export default class ClinicalTable extends React.Component<IClinicalTableProps,
         return _.sumBy(this.props.data, obj => obj.count);
     }
 
+    @autobind
```

Comment:
not 100% sure but i don't think you need autobind on these.  you should only need autobind when a DOM callback is changing the context (this) when executing callback.    i.e. onclick={this.someCallback} unless you bind someCallback to the desired instance, it will be the clicked element.